pls_core/src/inflections/mod.rs

```diff
@@ -59,11 +60,11 @@ pub fn generate_inflection_table(
     host_version: &str,
     transliterate: fn(&str) -> Result<String, String>,
     exec_sql_query: fn(&str) -> Result<String, String>,
+    locale: &str,
 ) -> Result<String, String> {
     let q = SqlQuery::new(exec_sql_query);
     let pm = get_pali1_metadata(pali1, transliterate, &q)?;
-    let body = generators::create_html_body(&pm, transliterate, &q)?;
-
```

Comment:
these are unnecessary line deletions. not related to your change so should not be there.

if there is a reason for this, please have a separate refactoring commit

Reply:
I'll keep this in mind.